composer/core/state.py

```diff
@@ -336,6 +336,12 @@ def schedulers(self):
     def schedulers(self, schedulers: Union[types.PyTorchScheduler, Sequence[types.PyTorchScheduler]]):
         self._schedulers[:] = ensure_tuple(schedulers)
 
+    def batch_set_item(self, key: Optional[Any] = None, value: Optional[Any] = None, set_fn: Optional[Callable] = None):
```

Comment:
can we move up or duplicate the batch_set/batch_get docstrings to here? Most people implementing algorithms should use this functionality directly from the state object so we should make it easy for them to find out how it works.

Reply:
good idea. done here: [247f2bf](https://github.com/mosaicml/composer/pull/978/commits/247f2bf5f8665e93f0d8c29cd669f11dddb627b9)